src/components/App.jsx

```diff
@@ -36,6 +36,10 @@ import azureMapsExt from '../libs/azure-maps-ext';
 
 import MapboxGl from 'mapbox-gl'
 
+const levelOrdinalFilterExceptionLayers = ['indoor_level', 'indoor_facility'];
```

Comment:
Can you make the exception layers filtered by 'source-layer' name instead of layer id? Here is the list ['facility', 'facility_area', 'level', 'level_area']. Since other layers like indoor_facility_POI should also be filtered based on its source-layer.

Reply:
In the meantime, if you can also apply level filter for 'indoor_level_footprint_boundary' layer, that will be good, since this layer is also level-related.